packages/account/src/account.ts

```diff
@@ -500,6 +501,35 @@ export class Account extends AbstractAccount {
     return this.sendTransaction(request);
   }
 
+  async getTransactionCost(
+    transactionRequestLike: TransactionRequestLike,
+    { signatureCallback, quantitiesToContract = [] }: TransactionCostParams = {}
```

Comment:
The property name `quantitiesToContract` could use a renaming as it is not always used as quantities being forwarded to contracts. 

The Account method [withdrawToBaseLayer](https://github.com/FuelLabs/fuels-ts/blob/f3453b9ea669575e3cf97ee50512bc9039845e0c/packages/account/src/account.ts#L464) uses the property to inform the amount of the base asset that should be withdrawn to the base layer.

This is not a hard requirement but in my opinion, since we are already refactoring and breaking things, we could rename this property. What do you think?

Reply:
I agree, however, I wasn't sure of a suitable name. What about just `quantities`?

Reply:
Sure, this one is good as any

Reply:
- Fixed docs: 126107d7d6e69190c946a4ef20c2fc1b00537802
- Updated parameter: 4c81e4b4c0ebd11fddd1c3e3f326645b498534d9